scripts/appleseed.package.py

```diff
@@ -916,14 +936,29 @@ def get_dependencies_for_file(self, filepath):
             if len(line) == 0:
                 continue
 
+            # Ignore libs without any dyanmic dependencies.
+            if "statically linked" in line:
+                continue
+
             # Ignore system libs.
             if self.__is_system_lib(line):
                 continue
 
             libs.add(line.split()[2])
 
+        if libs:
+            trace("  Dependencies for {0}:".format(filepath))
+            for lib in libs:
+                trace("      {0}".format(lib))
+        else:
+            trace("  Dependencies for {0}: None".format(filepath))
+
         return libs
 
+    def __is_shared_lib(self, filename):
+        ext = os.path.splitext(filename)[1]
+        return ext == ".so" or ".so." in filename
```

Comment:
i propose the following : `filename.endswith(".so") || ".so" in filename`

Reply:
That's not quite correct, the second dot in `".so."` is important (otherwise it would pick things like `file.something.ext`).

Reply:
My bad, then it is 


`filename.endswith(".so") || ".so." in filename`

Reply:
Ok.